### ISSUE TITLE
Can I use this on Windows?

### ISSUE THREAD

**Issue by tyeeman** [arnaucube/blogo#2]
I see the main app called blogo with no file extension. Is this for linux only?

**Comment by arnaucube** [comment]
yes, the 'blogo' file is the binary compiled for Linux. But you can compile it for windows, and it will generate the file 'blogo.exe'

**Comment by tyeeman** [comment]
Great, but I'm not a coder. To compile it do I just install GO language, and then run a compiler command and it's done or do I have to know coding?
